helpers/hstring.py

```diff
@@ -94,6 +94,48 @@ def get_docstring_line_indices(lines: List[str]) -> List[int]:
     return docstring_line_indices
 
 
+# TODO(allenmatt10): Add tests.
+def get_function_indices(lines: List[str]) -> List[int]:
```

Comment:
lines -> txt

Let's say what it represents and now how it's encoded.
Can you change also the function below? (Which is not great)